dgraph/cmd/alpha/run.go

```diff
@@ -263,8 +263,8 @@ they form a Raft group and provide synchronous replication.
 
 	flag.String("feature-flags", worker.FeatureFlagsDefaults, z.NewSuperFlagHelp(worker.FeatureFlagsDefaults).
 		Head("Feature flags to enable various experimental features").
-		Flag("list-in-normalize", "enables returning a list when there are multiple fields with same alias, "+
-			"see here for more details https://github.com/dgraph-io/dgraph/pull/7639").
+		Flag("normalize-response", "chose the kind of response for @normalize when there are multiple fields with same "+
+			"alias, could be `list` or `flat`. For more details, see https://github.com/dgraph-io/dgraph/pull/7639").
```

Comment:
```suggestion
		Flag("normalize-response", "configure @normalize formatting. "+
			" `list`:  fields with the same alias are returned as a list; 'flat': returned  a single value for fields  with same alias  (default behavior)").
```

Reply:
Could we be more precise on which single value is returned? Is it the first one, the deepest level, the first level ?
If we can then change "a single value" by the correct description.

Reply:
We return all of them, it it invalid json but has same key multiple times.